testnet/README.md

```diff
@@ -114,12 +128,11 @@ service.yaml somehow instead
 
 XXX TRY:
 kubectl apply -f ./service.yaml
-
 XXX BUT:
 i'm pretty sure the labels are wrong. I don't know what they are
 supposed to be for, but I don't see how the service can attach to the deployment.
 
-# XXX things dont work below here maybe
```

Comment:
revert this back? i think you squashed a change i made while you were making this